stdlib/src/lib.rs

```diff
@@ -133,6 +135,11 @@ pub fn get_module_inits() -> impl Iterator<Item = (Cow<'static, str>, StdlibInit
         {
             "_bz2" => bz2::make_module,
         }
+        // Windoes-only
+        #[cfg(windows)]
+        {
+            "win32api" => win32api::make_module,
+        }
```

Comment:
`_winapi.GetModuleFileName` is part of `_winapi` module. We don't need to add our original module for the purpose.